cmd/juju/machine/upgradeseries.go

```diff
@@ -245,16 +250,34 @@ func (c *upgradeSeriesCommand) UpgradeSeriesPrepare(ctx *cmd.Context) (err error
 		defer apiRoot.Close()
 	}
 
-	units, err := c.upgradeMachineSeriesClient.UpgradeSeriesValidate(c.machineNumber, c.series)
+	// get the units for a given machine.
+	fullStatus, err := c.statusClient.Status(nil)
```

Comment:
Can we break this out into a units method or the like?

Reply:
Yeah, sure don't mind doing that.